python35/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+VER=$(cat VERSION)
+
+docker build -t zalando/python:$VER .
+
+SQUASH_PATH=$(which docker-squash)
+
+if [ -z "$SQUASH_PATH" ]; then
+    # install Docker Squash from https://github.com/jwilder/docker-squash
+    TGZ=docker-squash-linux-amd64-v0.1.0.tar.gz
```

Comment:
There is a new Docker Squash version, see https://github.com/zalando/docker-ubuntu/blob/master/build.sh